lib/structures/Channel.js

```diff
@@ -19,6 +19,30 @@ class Channel extends Base {
         return `<#${this.id}>`;
     }
 
+    static from(data, extra) {
```

Comment:
Looks like it would be better to pass `client` consistently, then have the `GuildChannel`s get the guild as needed.